Documentation/Storage-Configuration/Advanced/ceph-configuration.md

```diff
@@ -47,12 +47,21 @@ kubectl apply -f common.yaml -f operator.yaml -f cluster.yaml # add other files
 
 ## Deploying a second cluster
 
-If you wish to create a new CephCluster in a different namespace than `rook-ceph` while using a single operator to manage both clusters execute the following:
+If you wish to create a new CephCluster in a different namespace than `rook-ceph` while using a
```

Comment:
If they want to use `rook-ceph` and `rook-ceph-secondary`, there is no need for the `sed` command. Maybe adjust this statement to say it's optional if they want to change those defaults?